superset/db_engine_specs/hive.py

```diff
@@ -332,49 +333,51 @@ def _partition_query(cls, table_name, limit=0, order_by=None, filters=None):
     @classmethod
     def select_star(
         cls,
-        my_db,
+        database,
         table_name: str,
         engine: Engine,
         schema: str = None,
         limit: int = 100,
         show_cols: bool = False,
         indent: bool = True,
         latest_partition: bool = True,
-        cols: List[dict] = [],
```

Comment:
This did not look right; if this was intentional, a comment should be present.

 https://blog.florimond.dev/python-mutable-defaults-are-the-source-of-all-evil